join/launchValidator.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/sh
+set -e
+
+AXELAR_CORE_VERSION=""
+TOFND_VERSION=""
+ROOT_DIRECTORY=~/.axelar_testnet
+
+for arg in "$@"; do
+  case $arg in
+    -r|--root)
+    ROOT_DIRECTORY="$2"
+    shift
+    ;;
+    --axelar-core)
+    AXELAR_CORE_VERSION="$2"
+    shift
+    ;;
+    --tofnd)
+    TOFND_VERSION="$2"
+    shift
+    ;;
+    *)
+    shift
+    ;;
+  esac
+done
+
+if [ -z "$AXELAR_CORE_VERSION" ]; then
+  echo "'--axelar-core vX.Y.Z' is required"
+  exit 1
+fi
+
+if [ -z "$TOFND_VERSION" ]; then
+  echo "'--tofnd vX.Y.Z' is required"
+  exit 1
+fi
+
+if [ ! -d "$ROOT_DIRECTORY" ]; then
+  echo "Root directory does not exist"
+  exit 1
+fi
+
+SHARED_DIRECTORY="${ROOT_DIRECTORY}/shared"
+if [ ! -d "$SHARED_DIRECTORY" ]; then
+  echo "Shared directory does not exist"
+  exit 1
+fi
+
+NODE_UP="$(docker ps --format '{{.Names}}' | grep -w 'axelar-core')"
+if [ -z "$NODE_UP" ]; then
+  echo "No node running"
+  exit 1
+fi
```

Comment:
not sure if this is sufficent. Would this be triggered by a local cluster instance as well?